components/switch/style/index.less

```diff
@@ -8,7 +8,7 @@
   position: relative;
   display: inline-block;
   box-sizing: border-box;
-  height: 22px;
+  height: @input-height-base;
```

Comment:
Should not modify this:

<img width="207" alt="image" src="https://cloud.githubusercontent.com/assets/507615/21211472/c319607c-c2be-11e6-96cd-025c947e3e26.png">